gen/struct_test.go

```diff
@@ -316,5 +330,45 @@ func TestNestedStructsOptional(t *testing.T) {
 		if assert.NoError(t, s.FromWire(tt.v)) {
 			assert.Equal(t, tt.s, s)
 		}
+
+		assert.Equal(t, tt.o, tt.s.String())
+	}
+}
+
+func TestStringDoesNotPanicOnMissingRequiredFields(t *testing.T) {
+	// must not panic on String() even if required fields are missing.
```

Comment:
can rename test + remove comment, since if hte tests pass, it's not causing panics